.circleci/config.yml

```diff
@@ -43,17 +52,23 @@ workflows:
   version: 2
   update:
     jobs:
+     - eslint:
+        filters:
+          tags:
+            only: /[0-9].[0-9].[0-9]/
+          branches:
+            ignore: /.*/
       - build:
           filters:
```

Comment:
you should add the following before `filters`:
```
requires:
  - test
```

Reply:
you mean -eslint?

Reply:
yup